analysers/Analyser_Osmosis.py

```diff
@@ -204,6 +204,38 @@ class Analyser_Osmosis(Analyser):
 CREATE INDEX idx_multipolygons_poly_proj ON {0}.multipolygons USING gist(poly_proj);
 CREATE INDEX idx_multipolygons_tags ON {0}.multipolygons USING gist (tags);
 ANALYZE {0}.multipolygons;
+"""
+
+    sql_create_polygons = """
+CREATE UNLOGGED TABLE {0}.polygons AS
+SELECT
+    'R' AS type,
+    id,
+    tags,
+    poly,
+    poly_proj
+FROM
+    {0}.multipolygons
+WHERE
+    is_valid
+UNION ALL
+SELECT
+    'W' AS type,
+    id,
+    tags,
+    ST_MakePolygon(linestring) AS poly,
+    CASE WHEN ST_IsValid(ST_MakePolygon(ST_Transform(linestring, {1}))) THEN ST_MakePolygon(ST_Transform(linestring, {1})) ELSE NULL END AS poly_proj
```

Comment:
To avoid having to do a NULL check for the very rare case where it isn't true, perhaps `ST_IsValid(ST_MakePolygon(ST_Transform(linestring, {1})))`can just be in the WHERE

Reply:
I think this was resolved based on the commit I see below, so I'll close this